canonical-web-and-design/maas.io
# Add lazysizes to images

## Done

Add the lazysizes library and apply it too all images on main pages except the hero image on `/how-it-works`

## QA

- Go to the homepage, /install, /how-it-works and /tour
- See that all the images load
